radio/src/lua/api_general.cpp

```diff
@@ -498,6 +498,26 @@ static int luaGetValue(lua_State * L)
   return 1;
 }
 
+/*luadoc
+@function getRAS()
+
+Return the RAS value or nil in case of issue 
+
+@retval number representing RAS value or nil if if there is an hardware issue
+This is just a hardware pass/fail measure and does not represent the quality of the radio link
+
+@status current Introduced in 2.2.0
+*/
+static int luaGetRAS(lua_State * L)
+{
+  if (telemetryData.swr.value > 0x33) {
```

Comment:
I don't think the test is right, I would rather see:

```
IS_VALID_XJT_VERSION()
```

Well indeed this macro name is not very good ...